scripts/Modules/mkw_classes.py

```diff
@@ -583,7 +583,7 @@ def __init__(self, playerIdx=0):
         
     @staticmethod
     def instance(playerIdx=0):
-        offsets = [0x20, playerIdx * 0x4, 0x10, 0x10, 0x1C]
+        offsets = [0x20, playerIdx * 0x4, 0x10, 0x1C]
```

Comment:
There's a faster one.
```suggestion
        offsets = [0x20, playerIdx * 0x4, 0x20]
```

Reply:
I think this assumes that KartObject and KartState are allocated next to each other. Are you sure this is safe?

Reply:
I'm confident this is safe, and can pull relevant code explaining why in around 10 minutes.

Reply:
Works on my end... HOWEVER, I do want us to keep in mind that when we reorganize `mkw_classes.py` to include subclasses and a proper hierarchal representation of all these classes, will it really make sense to retrieve `KartState` with this pointer chain?

Reply:
> Will it really make sense to retrieve `KartState` with this pointer chain?

Yes, because it is also the way we access a bunch of other classes (including a chain for the `VehiclePhysics` instance, of which no other method exists).